server/bin/pbench-server-prep-shim-002.sh

```diff
@@ -80,9 +80,7 @@ trap 'rm -rf $tmp' EXIT INT QUIT
 mkdir -p $tmp
 sts=$?
 if [ $sts != 0 ]; then
-    echo "Failed: \"mkdir -p $tmp\", status $sts" >> $errlog
-    log_finish
-    exit 4
+    log_exit "Failed: \"mkdir -p $tmp\", status $sts" >> $errlog
```

Comment:
It seems the original code was wrong here.  It should have being doing `echo "Failed: ..." >&4` since we have issued the `log_init` already at line 74.

So then this should probably become:

```
log_exit "Failed: \"mkdir -p $tmp\", status $sts" 4
```